_data/task.yml

```diff
@@ -60,23 +60,23 @@ websites:
     url: https://www.meistertask.com
     img: meistertask.png
     tfa:
-      - software
-    doc: https://support.meistertask.com/hc/en-us/articles/207318670
+      - totp
+    doc: https://support.mindmeister.com/hc/en-us/articles/218901367
```

Comment:
I guess you choose the MindMeister URL because it details the TOTP Apps?

Reply:
MeisterTask uses the MindMeister account, so the tfa method(s) should be the same.